students/seungjun/users/views.py

```diff
@@ -41,3 +41,23 @@ def post(self, request):
 
         except KeyError:
             return JsonResponse({'message':'KEY_ERROR'}, status=400)
+
+class LogInView(View):
+    def post(self,request):
+        data = json.loads(request.body)
+        try:
+            input_email    = data['email']
+            input_password = data['password']
```

Comment:
변수명은 간결하고 명확하게 선언해주세요. `input_` 과 같은 용어는 불필요합니다~
또한 json.loads() 함수를 호출하는 경우에 decode error가 발생할 수 있으므로 `try`, `except`문 안에 추가해주세요!

```suggestion

        try:
            data      = json.loads(request.body)
            email    = data['email']
            password = data['password']
```

Reply:
data 변수는 try문 안에서 선언하고, email과 password 변수명 수정하겠습니다